### PR TITLE
Update container for kegg pathways completeness tool

### PR DESCRIPTION
**Pull request by KateSakharova**
Added container based on kegg-pathways-completeness tool repo 
https://github.com/EBI-Metagenomics/kegg-pathways-completeness-tool/tree/master
Container pushed under `quay.io/microbiome-informatics/kegg-completeness:v1.1`

### CODE REVIEW

**Review by Ales-ibt — CHANGES_REQUESTED** [review]
Looks ok to me and the image works ok after I tested it. However, it would be great If `-o` option would have effect on the files names instead of on a directory name

**Comment by mberacochea** [comment]
We should, in the future, publish this in bioconda / biocontainers

File: kegg-pathways-completeness-tool/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.14
+
+LABEL Maintainer="Ekaterina Sakharova <kates@ebi.ac.uk>"
+
+# Install dependencies and Git
+RUN apk add --no-cache --update-cache --update python3 gzip git && \
+    apk add --no-cache --update py3-pip && \
+    apk add --no-cache --update bash && \
```
Comment (mberacochea) [inline kegg-pathways-completeness-tool/Dockerfile:8]: We need to install the nextflow required packages, I don't remember the list of them for alpine. I'll get them later

Comment (KateSakharova) [reply kegg-pathways-completeness-tool/Dockerfile:8]: true, I know what they are, will add

**Review by Ales-ibt — APPROVED** [review]
I've tested the container and works well with the recent changes. I'll use it on my nextflow pipelines and open an issue if find any problems.

**Comment by KateSakharova** [comment]
I've added necessary libs for Nextflow. sed and awk are already installed in alpine

**Pull request merged**